blog/test.py

```diff
@@ -11,11 +11,23 @@
 from trips import views
 
 
+def create_user():
+    user = User(email='emailtest@test.com',
```

Comment:
You could move this into the `setUp` of the `ArticleTest` class:
`self.user = User(…`